### PR TITLE
[Manta] Adapt to new pallet-manta-pay interface

### PR DESCRIPTION
**Pull request by stechu**
This PR adapt `pallet-manta-pay` version.

### CODE REVIEW

File: runtimes/manta/runtime/Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ authors = ['Manta Network']
 edition = '2018'
 license = 'GPL V3'
 name = 'manta-runtime'
-version = '3.0.0'
+version = '3.0.1'
```
Comment (Dengjianping) [inline runtimes/manta/runtime/Cargo.toml:6]: Do we only bump runtime version?

File: runtimes/manta/runtime/src/lib.rs [hunk 2]
```diff
@@ -107,8 +106,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
-	spec_version: 2,
-	impl_version: 1,
+	spec_version: 3,
+	impl_version: 2,
```
Comment (Dengjianping) [inline runtimes/manta/runtime/src/lib.rs]: In my opinion, most of time, we don't need to bump `impl_version`.
Reference: https://substrate.dev/docs/en/knowledgebase/runtime/upgrades

File: runtimes/manta/runtime/Cargo.toml (hunk 1 above)
Comment (stechu) [reply runtimes/manta/runtime/Cargo.toml:6]: yeah, we should bump other version as well. good call.

File: runtimes/manta/runtime/src/lib.rs (hunk 2 above)
Comment (stechu) [reply runtimes/manta/runtime/src/lib.rs]: agreed. will revert it back.

**Review by Dengjianping — APPROVED** [review]
LGTM!

**Pull request merged**